src/data/contributors/RitwikMehta.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Ritwik Mehta",
+  "skills": ["C++", "Flutter/Dart", "FullStack"],
+  "cause": "I code to build. To be able to teach one day.",
+  "image": "Avatar3",
+  "github": "https://github.com/Ritwikmehta28",
+  "twitter": "https://www.geekyhermit.com/",
```

Comment:
You need to specify link of your twitter account here if you wish to add or you can keep this blank.